challenge_3/python/slansford/src/challenge_3.py

```diff
@@ -0,0 +1,7 @@
+given_array = [2,2,3,7,5,7,7,7,4,7,2,7,4,5,6,7,7,8,6,7,7,8,10,12,29,30,19,10,7,7,7,7,7,7,7,7,7] 
+
+for i in given_array:
+	if given_array.count(i) > len(given_array)/2:
+		print(i)
+		break
```

Comment:
Same issue here as in challenge 2, with the count() inside of a loop. Can be optimized.

Also, note that `len(given_array)/2` isn't _equal_ to the floor of the length divided in half. You can either use a floor function or do integer division. Assuming that `given_array.count(i)` is an integer, though, this shouldn't matter too much.